home_audit.node.js

```diff
@@ -673,17 +673,12 @@ let validationRules = {
     },
 
     skylight_area_2: function(value) {
```

Comment:
These second skylight rules were added because (1) they exist in the api and (2) they are headers in the csv batch import, and thus need validation to keep everything for bursting into flames.  Should these be moved to the csv validation rules, or should they just be kept here with other legitimate home values?